tasks/us/census/spielman_singleton_segments.py

```diff
@@ -23,38 +23,33 @@ def filename(self):
         return 'understanding-americas-neighborhoods-using-uncertain-data-from-the-american-community-survey-output-data.zip'
 
     def url(self):
-        return 'https://www.openicpsr.org/repoEntity/download'
+        return 'https://www.openicpsr.org/openicpsr/project/100235/version/V5/download/project?dirPath=/openicpsr/100235/fcr:versions/V5/Output-Data'
 
     def curl_request(self):
-        docid = 41528
-        email = 'slynn@cartodb.com'
-        return 'curl -L {url} --data "downloaderEmail={email}&agree=I Agree&id={docid}"'.format(
-            docid = docid,
-            email=email,
-            url= self.url());
+        return 'curl -L {url}'.format(url=self.url())
```

Comment:
Why don't we use requests instead of calling curl and checking for system output?

Reply:
Even create a helper to download files using requests, if we don't have one, to be used as "our internal API"

Reply:
We need to have a working ETL ASAP. I have created a new issue (#301) to solve this

Reply:
Great :)